pandas/_libs/tslibs/dtypes.pxd

```diff
@@ -18,6 +18,7 @@ cdef dict c_DEPR_ABBREVS
 cdef dict attrname_to_abbrevs
 cdef dict npy_unit_to_attrname
 cdef dict attrname_to_npy_unit
+cdef set c_REMOVED_ABBREVS
```

Comment:
I don't think we need to keep track of these over time. Should just be able to remove from `c_DEPR_ABBREVS` and call it a day I think